app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
As a side note, it's worth noting that if the `input` is not _in_ the `label`, then one needs to mark the relation up through `id` and `for` attributes… which is kind of heavy.